crates/storage/codecs/src/lib.rs

```diff
@@ -26,6 +26,8 @@ use bytes::{Buf, BufMut};
 extern crate alloc;
 #[cfg(not(feature = "std"))]
 use alloc::vec::Vec;
+#[cfg(feature = "std")]
+use std::vec::Vec;
```

Comment:
is this required?
this should be in scope if std is enabled?

Reply:
I decided to ad this and use `crate::Vec` everywhere else to keep PR minimal.

Otherwise in all other place it will be like this;

```
#[cfg(feature = "std")]
use std::vec::Vec;
#[cfg(not(feature = "std"))]
use alloc::vec::Vec;
```

Which is 4 lines. But maybe this is clearer. 

What do you think?

Reply:
Changed